### PR TITLE
Update ORT Mobile example [Android Image Classifier]

### PR DESCRIPTION
**Pull request by gwang-msft**
Update mobile example to reflect the latest changes in the ORT Mobile
- Added the link to the jupyter notebook (PyTorch -> ONNX -> Quantization -> Convert to ORT), the notebook still need some pending updates (add convert to ORT, https://github.com/microsoft/onnxruntime/pull/7941)
- Used ORT mobile package from maven central, remove custom build part of the example
- Reorganized the Readme
- Minor updates to the App

### CODE REVIEW

File: mobile/examples/image_classifications/android/README.md [hunk 1]
```diff
@@ -1,46 +1,42 @@
 # ONNX Runtime Mobile image classification Android sample application
 
 ## Overview
-This is an example application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android. The demo app uses image classification which is able to continuously classify the objects it sees from the device's camera in real-time and displays the most probable inference result on the screen.
+This is an example application for [ONNX Runtime](https://github.com/microsoft/onnxruntime) on Android. The demo app uses image classification which is able to continuously classifying the objects it sees from the device's camera in real-time and displays the most probable inference results on the screen.
```
Comment (skottmckay) [inline mobile/examples/image_classifications/android/README.md:4]: >which is able to continuously classifying the objects it sees from the device's camera in real-time and displays the most probable inference  [](http://example.com/codeflow?start=143&length=141)

'continuously classify' was correct. 

or

'The demo app uses image classification that continuously classifies...'

File: mobile/examples/image_classifications/android/README.md [hunk 2]
```diff
@@ -1,46 +1,42 @@
...
-The above three files are provided and can be downloaded [here](https://1drv.ms/u/s!Auaxv_56eyubgQX-S_kTP0AP66Km?e=e8YMX1).
-
-[Optional] You can also build your own ONNX Runtime arm64 AAR files for Android. (See [build instructions here](https://www.onnxruntime.ai/docs/how-to/build.html#android) and [Build Android Archive(AAR)](https://www.onnxruntime.ai/docs/how-to/build.html#build-android-archive-aar)).
+### Step 0. [Optional] Prepare the ORT models
+Open [Mobilenet v2 Quantization with ONNX Runtime Notebook](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/notebooks/imagenet_v2/mobilenet.ipynb), this notebook will demostrate how to,
+1. Export the pre-trained MobileNet V2 FP32 model from PyTorch to a FP32 ONNX model
+2. Quantize the FP32 ONNX model to an uint8 ONNX model
+3. Convert both FP32 and uint8 ONNX models to ORT models
 
+Note: this step is optional, you can download the FP32 and uint8 ORT models [here](https://1drv.ms/u/s!Auaxv_56eyubgQlfGIBWh-j1wYyl?e=GYKoL7).
```
Comment (skottmckay) [inline mobile/examples/image_classifications/android/README.md:26]: If it's easy enough to just run the notebook I don't think we should provide a download as the ORT models may differ between ORT versions in the future.

Comment (gwang-msft) [reply mobile/examples/image_classifications/android/README.md:26]: The quantized model from the notebook is quantized using some example images, not based on a proprietary calibration set, so the model will have accuracy issue.
This note book is showing how to export pytorch model -> quantize -> convert to ort, but will not generate a working quantized model.

Comment (skottmckay) [reply mobile/examples/image_classifications/android/README.md:26]: OK. For now it's ok but we may want to point to a landing page in the docs where they can download models for a particular ORT version (e.g. we add new optimizers or contrib ops or the ORT model format version changes for some reason).

**Review by skottmckay — APPROVED** [review]
:shipit:
